packages/react-components/react-dialog/library/package.json

```diff
@@ -29,6 +29,7 @@
     "@fluentui/keyboard-keys": "^9.0.8",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33288&commitId=08b6ee018f5106074a7db8aa5ce90fb9c7837ac5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|1| Changed|
|Avatar Converged.badgeMask.chromium.png|36| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2231| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>